### PR TITLE
fix: Coj contents details

### PR DESCRIPTION
**Pull request by DorisWongNHS**

### CODE REVIEW

File: styles/globals.scss [hunk 1]
```diff
@@ -374,3 +374,7 @@ button:disabled {
     display: table-cell !important;
   }
 }
+
+ol[type="a"] {
```
Comment (john12321) [inline styles/globals.scss:378]: I like this @DorisWongNHS !

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "trainee-ui-app",
-  "version": "0.65.1",
+  "version": "0.65.3",
```
Comment (john12321) [inline package.json:3]: this might be a typo but no big deal.

**Review by john12321 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (DorisWongNHS) [reply package.json:3]: I am not sure the order we want to merge it. But @Judge40's PR is having `0.65.2`
https://github.com/Health-Education-England/tis-trainee-ui/pull/860/files

Comment (john12321) [reply package.json:3]: Ah OK, I think you may have got there first! 😄

**Review by ReubenRobertsHEE — DISMISSED** [review]

**Review by john12321 — APPROVED** [review]

**Pull request merged**